generator/views/operational_monitoring_view.py

```diff
@@ -0,0 +1,95 @@
+"""Class to describe an Operational Monitoring View."""
+
+from textwrap import dedent
+from typing import Any, Dict, List, Optional, Set
+
+from . import lookml_utils
+from .ping_view import PingView
+from .view import ViewDict
+
+# These are fields we don't need in our view
+EXCLUDED_FIELDS: Set[str] = {
+    "submission",
+    "client_id",
```

Comment:
Any chance we can get `sample_id` in here? That is a nice one to make available for faster iterative analysis.

Reply:
You mean to exclude `sample_id` or to include it? This is the exclude list. The table I'm prototyping with in BQ doesn't have a `sample_id` at the moment but this would require a change on the bq_etl side when a json config for operational monitoring is processed, then `sample_id` should be added in if it exists. I filed an issue for this change: https://github.com/mozilla/bigquery-etl/issues/2188

Reply:
Yes sorry - I commented here because I saw `client_id`. We should include sample_id.